File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/buttongroup/vb/guids.vb
'-----------------------------------------------------------------------------
' Copyright (c) Microsoft Corporation.  All rights reserved.
'-----------------------------------------------------------------------------
' Guids.vb
' MUST match guids.h

Namespace Microsoft.Create_HandleCommands
	Friend NotInheritable Class GuidList
		Public Const guidCreate_HandleCommandsPkgString As String = "2813fc1f-3b2c-4bb5-a166-c35d38508171"
		Public Const guidCreate_HandleCommandsCmdSetString As String = "a355053a-8a80-4142-98bc-9a7b0e14eee2"

		Public Shared ReadOnly guidButtonGroupCmdSet As New Guid(guidCreate_HandleCommandsCmdSetString)
		Public Shared ReadOnly guidMenuTextCmdSet As New Guid(guidCreate_HandleCommandsCmdSetString)

	End Class
End Namespace

File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/buttongroup/cs/guids.cs
// Guids.cs
// MUST match guids.h
using System;

namespace Microsoft.ButtonGroup
{
    static class GuidList
    {
        public const string guidButtonGroupPkgString = "54bd8472-ad28-4211-8dd0-250ead07ed41";
        public const string guidButtonGroupCmdSetString = "f69209e9-975a-4543-821d-1f4a2c52d737";

        public static readonly Guid guidButtonGroupCmdSet = new Guid(guidButtonGroupCmdSetString);
    };
}